pystac/catalog.py

```diff
@@ -179,7 +179,7 @@ def clear_children(self):
         Return:
             Catalog: Returns ``self``
         """
-        self.links = [l for l in self.links if l.rel != 'child']
```

Comment:
Interesting that the linter is picking these up now as opposed to before through local and travis. Wonder if this rule was introduced in a later version? I have flake8 3.7.8 and 0.28.0 on my machine.

Reply:
After looking at this closer, it looks like this was caused by a mistake on my part. I was following the guidance of my editor, which was using a newer version of `flake8` (3.8.3).

Most of the issues were related to [E741](https://www.flake8rules.com/rules/E741.html). I think two reasonable paths forward are:

1. Bump `flake8` in `requirements-dev.txt` to 3.8+ and keep the adjustments in.
2. Revert the adjustments and keep the dependencies as-is.

I'm in favor of #1 because it seems like a reasonable role, and updating `flake8` makes sense. What do you think?

Reply:
I agree, #1 makes sense, would be best upgrade and the lint changes are improvements.